persistence/remote/src/test/resources/remote-cl-uri-config.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<infinispan
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="urn:infinispan:config:${infinispan.core.schema.version} https://infinispan.org/schemas/infinispan-config-${infinispan.core.schema.version}.xsd
+                          urn:infinispan:config:store:remote:${infinispan.core.schema.version} https://infinispan.org/schemas/infinispan-cachestore-remote-config-${infinispan.core.schema.version}.xsd"
+      xmlns="urn:infinispan:config:${infinispan.core.schema.version}"
+      xmlns:remote="urn:infinispan:config:store:remote:${infinispan.core.schema.version}" >
+
+   <!-- Default cache named to preserve old default cache name -->
+   <cache-container default-cache="RemoteStoreConfigUriTest">
```

Comment:
It should be possible to remove the `default-cache` attribute as we always retrieve the `RemoteStoreConfigUriTest` directly by name.

Reply:
Its not possible to remove default cache from the configuration as while fetching DefautCacheManager checks for default cache.

Reply:
Sorry I forgot about the call to `cm.getCache()`. If you update this to `cm.getCache(storeCacheName)` it should be possible to remove it.